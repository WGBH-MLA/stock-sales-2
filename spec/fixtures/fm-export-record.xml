<ROW MODID="0" RECORDID="0">
    <artesia_id>art-98786543210</artesia_id>
    <aspect_ratio>4:3</aspect_ratio>
    <barcode>Barcode1234</barcode>
    <category>Locations</category>
    <Ci_ID>ci-98786543210</Ci_ID>
    <clip_description>La Habana Vieja</clip_description>
    <clip_keywords>gothic architecture</clip_keywords>
    <clip_title>gothic building</clip_title>
    <codec>MP1234</codec>
    <digital_wrapper>MOV</digital_wrapper>
    <dimensions>1x2</dimensions>
    <drive_name>ABC</drive_name>
    <episode_number>123</episode_number>
    <event_date>08/09/2000</event_date>
    <event_location>Havana, Cuba</event_location>
    <file_name>test.mp4</file_name>
    <folder_name>folder</folder_name>
    <format>NTSC</format>
    <frames_per_second>30</frames_per_second>
    <id>GBH000123</id>
    <length>00:00:30;00</length>
    <origin>mysql</origin>
    <program>Gratuitous Explosions</program>
    <series>NOVA</series>
    <tape_number>1234</tape_number>
    <time_in>00:00:00;00</time_in>
    <type>Beta SP</type>
</ROW>