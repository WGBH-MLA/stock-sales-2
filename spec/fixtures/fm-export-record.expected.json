{
  "artesia_id": "art-98786543210",
  "aspect_ratio": "4:3",
  "barcode": "Barcode1234",
  "category": "Locations",
  "ci_id": "5cf8d001d37241a59d07d594010bdab9",
  "clip_description": "La Habana Vieja",
  "clip_keywords": "gothic architecture",
  "clip_title": "gothic building",
  "codec": "MP1234",
  "digital_wrapper": "MOV",
  "dimensions": "1x2",
  "drive_name": "ABC",
  "episode_number": "123",
  "event_date": "08/09/2000",
  "event_location": "Havana, Cuba",
  "file_name": "test.mp4",
  "folder_name": "folder",
  "format": "NTSC",
  "frames_per_second": "30",
  "id": "GBH000123",
  "length": "00:00:30;00",
  "origin": "mysql",
  "program": "Gratuitous Explosions",
  "series": "NOVA",
  "tape_number": "1234",
  "time_in": "00:00:00;00",
  "type": "Beta SP",
  "thumb_src": "https://d14nxlyhj878kl.cloudfront.net/cifiles/5cf8d001d37241a59d07d594010bdab9/thumbnail.jpg?Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiKjovL2QxNG54bHloajg3OGtsLmNsb3VkZnJvbnQubmV0L2NpZmlsZXMvNWNmOGQwMDFkMzcyNDFhNTlkMDdkNTk0MDEwYmRhYjkvdGh1bWJuYWlsLmpwZyIsIkNvbmRpdGlvbiI6eyJJcEFkZHJlc3MiOnsiQVdTOlNvdXJjZUlwIjoiMC4wLjAuMC8wIn0sIkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoxNTA3MzEwNjg2fX19XX0_&Signature=GaVW48IHtUXmf8l6zJIy-KIh4sWHU3pWVBWU~If3Z8XnXnfKjfbleJRwvygXfOOgpp4CK6Fch5-RNTLEBSxqWhRV1DUv2jHqQ3aslWuT9E0dQBTy9WVLYOGoi2BEh8cRDRWaMgFYmQDhbRdU35R3IqNZAgxiHvpsqGbA3ymPFXA_&Key-Pair-Id=APKAJNUSFH4IKQRZ44WA"
}